app/__tests__/app.test.js

```diff
@@ -122,6 +81,42 @@ describe("GET /articles/:article_id", () => {
     })
 })
 
+describe("GET /api/articles", () => {
+    describe("-- functionality tests", () => {
+        test("200: responds with 200 status code and all articles, articles should be in descending date order, have a comment count and no body key", () => {
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({body}) => {
+                const newestArticle = {
```

Comment:
have a look at the jest-sorted library to test whether the articles are sorted in the correct order